package.json

```diff
@@ -8,140 +8,45 @@
   },
   "scripts": {
     "lint": "solium --dir ./contracts",
-    "build": "webpack -p",
-    "truffle": "./node_modules/.bin/truffle",
     "test": "truffle migrate && truffle test --network test",
-    "dev": "webpack-dev-server -d --inline --hot",
     "coverage": "./node_modules/.bin/solidity-coverage",
     "build-contracts": "sol-merger './contracts/*.sol' ./build",
     "solgraph": "solgraph contracts/Conference.sol  > doc/Conference.dot && dot -Tpng doc/Conference.dot -o doc/Conference.png"
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/makoto/blockparty.git"
+    "url": "git+https://github.com/noblocknoparty/blockparty.git"
```

Comment:
Please fix repo name, and in below parts too

Reply:
Good catch